src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -110,6 +111,48 @@ static uint16_t               eq_count_max;
 static uint16_t               eq_count;
 static uint16_t               eq_idx;
 
+/* the number of EQs for aio contexts. (aio_eq_count + eq_count) <= eq_count_max */
+static uint16_t               aio_eq_count;
+
+struct aio_eq {
```

Comment:
A `d` or similar prefix for this struct would make it more obvious that this is something local when it's declared later on.